### ISSUE TITLE
Allow to reinstate a revoked key

### ISSUE THREAD

**Issue by un33k** [kylemanna/docker-openvpn#241]
Currently, it is not possible to reinstate a revoked key .. let's say you revoked the wrong client by mistake ... or any other situation you might have.

It would be great to minimally allow the same client (name) to be created if the old one was revoked for any reason.

**Comment by buchdag** [comment]
You can issue a new certificate for an existing CN if the previous certificate has been revoked **and** if you remove the old certificate's `.crt`, `.key` and `.req` files from your container's pki.

```
CLIENT="your_client_name"
OVPN="your_container_name"
docker exec -it "$OVPN" rm "/etc/openvpn/pki/issued/$CLIENT.crt"
docker exec -it "$OVPN" rm "/etc/openvpn/pki/private/$CLIENT.key"
docker exec -it "$OVPN" rm "/etc/openvpn/pki/reqs/$CLIENT.req"
```

#251 add a script that can optionally remove those files when you revoke a certificate.

If the files are removed, you'll be able to generate a new certificate : 

`docker exec -it "$OVPN" easyrsa  build-client-full "$CLIENT"`

Please note that this will generate a **new** certificate and therefore a **new** .ovpn configuration file you'll have to use for this client. You can't **reinstate** a revoked certificate using easyrsa, because allowing to reinstate revoked certificates is pretty much the opposite of the way pkis are supposed to work.

There is probably a way to do it anyway if you hack through easyrsa files, but you might also end up with a broken pki.

**Comment by kylemanna** [comment]
Closing in favor of solution merged in #251
